diff --git a/node_modules/loader-utils/lib/getHashDigest.js b/node_modules/loader-utils/lib/getHashDigest.js
index 45e340e..d9b9d30 100644
--- a/node_modules/loader-utils/lib/getHashDigest.js
+++ b/node_modules/loader-utils/lib/getHashDigest.js
@@ -39,11 +39,29 @@ function encodeBufferToBase(buffer, base) {
   return output;
 }
 
+// Note: Added the `loadCrypto()` function to override the `crypto` module's `createHash()` method.
+// If the "md4" algorithm gets passed to the `createHash()` method, then call the `createHash()` method with the "md5" algorithm instead.
+function loadCrypto() {
+    const c = require("crypto");
+    const originalCreateHash = c.createHash;
+
+    c.createHash = (...args) => {
+      if (args[0] == "md4") {
+        return originalCreateHash("md5");
+      }
+
+      return originalCreateHash(...args);
+    };
+
+    return c;
+  }
+
 function getHashDigest(buffer, hashType, digestType, maxLength) {
   hashType = hashType || 'md4';
   maxLength = maxLength || 9999;
 
-  const hash = require('crypto').createHash(hashType);
+  // Note: Replaced `require("crypto")` with `loadCrypto()`.
+  const hash = loadCrypto().createHash(hashType);
 
   hash.update(buffer);
 
